src/styles/_datavisyn.scss

```diff
@@ -28,24 +28,33 @@ $dataviyn_font: Monda !default;
 
 nav.navbar,
 nav.navbar-inverse {
+  min-height: auto; // Temp fix until https://github.com/phovea/phovea_clue/issues/140 is fixed
```

Comment:
It is fixed with https://github.com/phovea/phovea_ui/pull/69 and already merged into develop.